tests/unit_tests/example_test/Makefile.am

```diff
@@ -0,0 +1,13 @@
+AUTOMAKE_OPTIONS = foreign
+
+check_PROGRAMS = testdriver testdriver2
```

Comment:
More descriptive names would be good.  `make check` now reports 'PASS: testdriver', which is not very informative.